spec/models/spree/variant_spec.rb

```diff
@@ -1,3 +1,4 @@
 RSpec.describe Spree::Variant, type: :model do
   it { is_expected.to have_many(:volume_prices) }
+  it { is_expected.to have_many(:model_volume_prices) }
```

Comment:
I'm fine with these assertions if people prefer them, but I contest that they don't provide any value.

Testing that a relationship is defined and exists is superfluous to testing methods that _use_ that relationship. Which makes this redundant.